.expeditor/publish-release-notes.sh

```diff
@@ -24,9 +24,15 @@ pushd ./automate.wiki
 ## Improvements
 -
 
+## Compliance Profiles
```

Comment:
Should this perhaps be "Compliance Profile Updates" or is that obvious to those who would consume this info? (I am not amongst that group so I truly do not know. 😁 )